Update two sections in Active Directory 201 page

davops opened the pull request:
Add the sections for the Global Catalog and the 5 FSMO roles

miketheman commented:
Great start!
Please read http://www.opsschool.org/en/latest/meta/style_guide.html#spacing section on starting sentences on new lines.

There also appear to be stylized quotation marks - please replace with a normal ASCII `'`

davops commented:
I moved all the lines in a paragraph to their own lines and fixed the two apostrophes that were stylized. If there's anything else that needs to be reformatted, just let me know.

miketheman commented on active_directory_201.rst:
```diff
@@ -20,10 +20,81 @@ DHCP
+It can be used to search all objects in the forest.
+For example, when searching for the a specific user, the GC can be searched to find users not only in the current domain but also in other domains in the forest.
+The attributes included in the GC's subset of object attributes are those required by the schema and those most commonly used in searches.
+Attributes can be added and removed from the Global Catalog using the Active Directory Schema snap-in.
```
Is there any other mechanism to add/remove these? Powershell subcommands?

miketheman commented on active_directory_201.rst:
```diff
@@ -20,10 +20,81 @@ DHCP
 
-Advanced Active Directory Maintenance
-=====================================
+When Active Directory Domain Services is installed,  the GC is added to the first domain controller in the forest.
```
nitpick: double space after comma

miketheman commented on active_directory_201.rst:
```diff
@@ -20,10 +20,81 @@ DHCP
-Advanced Active Directory Maintenance
-=====================================
+When Active Directory Domain Services is installed,  the GC is added to the first domain controller in the forest.
+It can be added to additional domain controllers.
```
I would try to rewrite this and the following line, since the above is a statement of an option without "why", and the following explains the why.
Maybe remove the above, and after the below line, another one pointing the reader to the basics of "how".

miketheman commented on active_directory_201.rst:
```diff
@@ -20,10 +20,81 @@ DHCP
+Universal groups can contain objects from any domain in the forest.
+The GC holds the membership for all universal groups.
+As part of the authorization process, the authenticating DC retrieves the security identifier for all the member's security groups, which includes universal groups.
+Therefore, if the environment has more than one domain, users need to be able to access a global catalog server for authentication in order to find objects from the universal groups in other domains.
```
This could be followed with a reinforcing "and thereby not having to traverse the WAN..." or something similar to conclude the reasoning for multiple GCs.

davops replied:
The rest of these comments were addressed in the last commit. For this one though, I wanted to clarify. The point for that last sentence was not that users need to traverse the WAN. It is that the GC's role in authentication makes a difference in a multi-domain environment but wouldn't matter in a single domain forest (where universal groups would not include objects from another domain). I reworded it slightly to try to better explain that aspect.

miketheman commented on active_directory_201.rst:
```diff
@@ -20,10 +20,81 @@ DHCP
+This role is also forest-wide.
+It controls changes to the forest namespace.
+It can add, remove, rename, and move domains.
+Since domain names changes are infrequent, this role is rarely used.
```
"domain names changes" should be "domain name changes" or "domain names change infrequently"

miketheman commented on active_directory_201.rst:
```diff
@@ -20,10 +20,81 @@ DHCP
+
+This role is domain-wide.
+PDC stands for primary domain controller.
+It holds the latest passwords and lockout information for all accounts.
```
Passwords, or password hashes?

miketheman commented on active_directory_201.rst:
```diff
@@ -20,10 +20,81 @@ DHCP
+PDC stands for primary domain controller.
+It holds the latest passwords and lockout information for all accounts.
+Password changes are immediately forwarded to the PDC from other DCs.
+Also, it is the time source for the domain.
```
"time source" -> "authoritative time source"

miketheman commented on active_directory_201.rst:
```diff
@@ -20,10 +20,81 @@ DHCP
+Since domain names changes are infrequent, this role is rarely used.
+By default, the role is held on the first server promoted to a DC in the forest.
+
+PDC Emulator
```
This section explains a lot about PDC, but not why there's an Emulator. Can you add something to explain that?

miketheman commented:
@davops Thanks so much for working through this! It's been merged and should show up in minutes.
🎉 👍
